apiserver/charmrevisionupdater/updater.go

```diff
@@ -145,7 +146,7 @@ func retrieveLatestCharmInfo(st *state.State) ([]latestCharmInfo, error) {
 		return nil, err
 	}
 
-	results, err := charmstore.LatestCharmInfo(client, curls)
+	results, err := charmstore.LatestCharmInfo(client, curls, "stable")
```

Comment:
Unfortunately, there will be a mix of channels.  So you'll need to change charmstore.LatestCharmInfo to take a []charm.ChannelURL (or equivalent) where each item has the channel hard-coded to "stable".